src/content/1.7/development/page-reference/back-office/order/view-order/_index.md

```diff
@@ -0,0 +1,86 @@
+- Edit product quantity - modify the quantity of the `OrderDetail` related to the selected product. See [OrderController::updateProductAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Add discount - create a new `CartRule` and assign it to this order. See [OrderController::addCartRuleAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+
+{{% notice %}}
+It is possible to have multiple invoices related to same order, therefore when editing a product or adding a discount you can select which invoice to use.
+{{< figure src="./img/order-invoice-select.png" title="Order invoice select" >}}
+{{% /notice %}}
+
+#### Messages block
+{{< figure src="./img/order-messages-bo.png" title="Order messages in Back Office" >}}
+
+This block shows order messages. These messages can be visible for the customer in Front Office order details or can stay hidden - that depends on the checkbox `Display to customer`.
+{{< figure src="./img/order-messages-fo.png" title="Order messages in Front Office" >}}
+Predefined message can be chosen in a dropdown which contains a list of message templates from `Sell -> Customer service -> Order messages`. This page can be quickly reached by clicking a shortcut link `Configure predefined messages`. Message sending is handled by [OrderController::sendMessageAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php), while the related javascript code is located in [order-view-page-messages-handler.js](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/admin-dev/themes/new-theme/js/pages/order/message/order-view-page-messages-handler.js).
+
+#### History block
+{{< figure src="./img/history-block.png" title="Order history block" >}}
+
+This block doesn't have an actual title like others, but let's just call it "History block" for now - it's the one containing following tabs:
+- Status - you can see transitions of this order statuses and update the status, you can also add an Order note in this tab. (`Order->note`). See [OrderController::setInternalNoteAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Documents - this tab contains invoices list.
+- Carriers - contains carriers information. You can change the current carrier and add a tracking number. See [OrderController::updateShippingAction](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php).
+- Merchandise returns - information about refunds history.
```

Comment:
Maybe we could quickly explain how modules can customize this part? Related to page https://devdocs.prestashop.com/1.7/modules/sample-modules/order-pages-new-hooks/